skyrim-platform/src/tilted/ui/TextToDraw.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+#include <array>
+#include <string>
+
+struct TextToDraw
+{
+  float x = 0.f;
+  float y = 0.f;
+  std::wstring kString;
```

Comment:
kString => string

According to https://github.com/skyrim-multiplayer/skymp/blob/main/docs/docs_cpp_code_guidelines.md#variables

> The first word is lowercase, any other words must start with an uppercase: camelCase.

> Global, static and thread-local variables must be prefixed with g_: static int g_foo;, unless they are constants.

> Constants should be prefixed with k. When declaring constants, please keep in mind that there can be side effects.